include/fastdds/dds/topic/ContentFilteredTopic.hpp

```diff
@@ -85,10 +130,6 @@ class ContentFilteredTopic : public TopicDescription
      */
     virtual DomainParticipant* get_participant() const override;
```

Comment:
```suggestion
    DomainParticipant* get_participant() const override;
```

`override` is enough to indicate that the method is `virtual`.

Also, this method is not being exported in Windows (`RTPS_DllAPI`) neither in the base clase (`TopicDescription`) nor here, so it is only part of the public API in non-Windows platforms. I think we should fix and export this method if it is intended to be used by the user and if not, remove the doxygen documentation so it does not appear in the API reference (and consider making the method protected).

Reply:
Removed the virtual on 55c498d, will do another PR adding the DLL export.

Reply:
See #2611